mobile-browser-based-version/src/components/Building Frames/ImageTestingFrame.vue

```diff
@@ -0,0 +1,380 @@
+      >
+        Test
+      </button>
+    </div>
+
+    <ImagePredictionResultFrame v-if="gotResults" :classes="classes" :imageElement="imgTested"/>
+
+    <div id="predictions"></div>
+
+    <!-- Upload Image Data Template-->
+    <template id="image-template">
+      <li class="block p-1 w-1/2 sm:w-1/3 md:w-1/4 lg:w-1/6 xl:w-1/8 h-24">
+        <article
+          tabindex="0"
+          class="group hasImage w-full h-full rounded-md focus:outline-none focus:shadow-outline bg-gray-100 cursor-pointer relative text-transparent hover:text-white shadow-sm"
+        >
+          <img
+            alt="upload preview"
+            class="img-preview w-full h-full sticky object-cover rounded-md bg-fixed"
+          />
+
+          <section
+            class="flex flex-col rounded-md text-xs break-words w-full h-full z-20 absolute top-0 py-2 px-3"
+          >
+            <h1 class="flex-1"></h1>
+            <div class="flex">
+              <span class="p-1">
+                <i>
+                  <svg
+                    class="fill-current w-4 h-4 ml-auto pt-"
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="24"
+                    height="24"
+                    viewBox="0 0 24 24"
+                  >
+                    <path
+                      d="M5 8.5c0-.828.672-1.5 1.5-1.5s1.5.672 1.5 1.5c0 .829-.672 1.5-1.5 1.5s-1.5-.671-1.5-1.5zm9 .5l-2.519 4-2.481-1.96-4 5.96h14l-5-8zm8-4v14h-20v-14h20zm2-2h-24v18h24v-18z"
+                    />
+                  </svg>
+                </i>
+              </span>
+
+              <p class="p-1 size text-xs"></p>
+              <button
+                class="delete ml-auto focus:outline-none hover:bg-gray-300 p-1 rounded-md"
+              >
+                <svg
+                  class="pointer-events-none fill-current w-4 h-4 ml-auto"
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="24"
+                  height="24"
+                  viewBox="0 0 24 24"
+                >
+                  <path
+                    class="pointer-events-none"
+                    d="M3 6l3 18h12l3-18h-18zm19-4v2h-20v-2h5.711c.9 0 1.631-1.099 1.631-2h5.316c0 .901.73 2 1.631 2h5.711z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </section>
+        </article>
+      </li>
+    </template>
+  </div>
+</template>
+
+
+<script>
+//TODO: add image example
+import ImagePredictionResultFrame from '../Building Frames/ImagePredictionsResultFrame'
+export default {
+  components: {
+    ImagePredictionResultFrame
+  },
+  props: {
+    Id: String,
+    Task: Object
+  }, 
+  data() {
+    return {
+      title: "mnist-testing",
+      DataFormatInfoText:"",
+      DataExampleText: "",
+      // Different Task Labels
+      task_labels: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+      IMAGE_HEIGHT: 28,
+      IMAGE_WIDTH: 28,
```

Comment:
Initialization from this.Task.training_information.IMAGE_HEIGHT, or something like that. 
I'm quite sure that once data() is initialized, the props have already been defined and thus can be accessed.